server/service/integration_vpp_install_test.go

```diff
@@ -944,11 +945,17 @@ func (s *integrationMDMTestSuite) TestVPPAppInstallVerification() {
 		s.Do("POST", "/api/latest/fleet/hosts/transfer",
 			&addHostsToTeamRequest{HostIDs: []uint{data.host.ID}, TeamID: &team.ID}, http.StatusOK)
 
+		// TODO(mna): until we have SCEP-based authentication for iDevices, cheat by
+		// inserting a token for that host so we can use the self-service API.
+		hostSecret := uuid.NewString()
+		mysql.ExecAdhocSQL(t, s.ds, func(q sqlx.ExtContext) error {
+			_, err := q.ExecContext(context.Background(), `INSERT INTO host_device_auth (host_id, token) VALUES (?, ?)`, data.host.ID, hostSecret)
+			return err
+		})
```

Comment:
Heads-up @cdcme , once we have the cert-based auth we should update this hack to use the proper auth in integration tests.

Reply:
I just reviewed your cert-auth PR, I can submit a follow-up with proper auth when yours lands, I'll extract a helper func from your tests.